CleverPush/Source/CPChatView.m

```diff
@@ -98,16 +98,49 @@ - (void)loadChatWithSubscriptionId:(NSString*)subscriptionId {
                 jsonConfig = [[NSString alloc] initWithData:jsonData encoding:NSUTF8StringEncoding];
             }
         }
-
-        NSString* brandingColor;
-        NSString* backgroundColor;
+        
+        NSString* brandingColor = @"";
```

Comment:
Can we please add all variables + methods directly in CPChatView class ?

That we do not have all these methods in our main class.

Reply:
Okay, we will make that change accordingly.